### PR TITLE
Import Support page from bazelbuild/bazel.

### LINKED ISSUE #19: Remove the Support page from the docs, move it to website

### PR DESCRIPTION
**Pull request by davidzchen**
Fixes #19

### CODE REVIEW

File: _includes/footer-content.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
       <p>About</p>
       <ul class="list-unstyled">
         <li><a href="https://github.com/bazelbuild/bazel/wiki/Bazel-Users">Who's using Bazel</a></li>
+        <li><a href="{{ site.main_site_url }}/support.html">Get Support</a></li>
```
Comment (steren) [inline _includes/footer-content.html]: We already have a support section in the footer. And the page is already linked (see Support Policy)

Comment (davidzchen) [inline _includes/footer-content.html]: Ah did not catch that. Done.

File: _includes/footer-content.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
         <li><a href="https://github.com/bazelbuild/bazel/issues">Issue Tracker</a></li>
         <li><a href="https://docs.bazel.build">Documentation</a></li>
         <li><a href="{{ site.main_site_url }}/faq.html">FAQ</a></li>
-        <li><a href="https://docs.bazel.build/support.html">Support Policy</a></li>
+        <li><a href="{{ site.main_site_url }}/support.html">Get Support</a></li>
```
Comment (steren) [inline _includes/footer-content.html]: Can you keep "Support Policy"? This was on purpose. the footer already contains how to Get support. the page is only useful for the Policy.

Comment (davidzchen) [inline _includes/footer-content.html]: Done.

**Pull request merged**